src/openvdb.imageio/openvdbinput.cpp

```diff
@@ -413,17 +442,18 @@ OpenVDBInput::readMetaData (const openvdb::GridBase& grid,
     }
 }
 
+
+
 bool
 OpenVDBInput::open (const std::string &filename, ImageSpec &newspec)
 {
     if (m_input)
         close();
 
     auto file = openVDB(filename, this);
-    if (!file)
+    if (!file || !file->isOpen())
```

Comment:
The `ASSERT` removed was because the contract from the `openVDB` function is not to return a `io::File` unless `isOpen` has returned true already.  In the scheme of what's about to be done the additional check likely adds nothing, but I did like the ASSERTION of that promise.

Reply:
I see. I was just trying to reduce the number of cases where we couldn't recover. From the point of an app, I think it's preferable to return "I can't open this file" whether it be that it's malformed, or OpenVDB is somehow broken, then to terminate the whole process. 

How about if I use DASSERT, which will do the hard crash/error when in debug mode? But for release build, the app will get a failure state rather than terminate?